client/src/screens/training/controller.js

```diff
@@ -12,18 +12,20 @@ export const controller = (serviceLocator) => {
       const params = getParams();
       const trainings = getData().trainings;
       const training = trainings.find(training => training.id === params.training);
+      if (typeof(training) !== 'undefined') {
```

Comment:
wow, I didn't know that typeof can be used like a function.
usually we use it like `typeof training !== 'undefined'`.
let's simplify this line and do the next:
```
if (training !== undefined) {
```

why?
because `training` is already undefined and is ready for comparison. no need for extra operation (checking its type)